README.md

```diff
@@ -1,237 +1,109 @@
+This repository hosts the following Node.js packages for API development at BYU OIT:
 
-[![codecov](https://codecov.io/gh/byu-oit/byu-jwt-nodejs/branch/main/graph/badge.svg?token=zOI4URNx3D)](https://codecov.io/gh/byu-oit/byu-jwt-nodejs)
+- [@byu-oit/jwt](./packages/jwt/README.md)
+- [@byu-oit/fastify-jwt](./packages/fastify/README.md)
 
-**Requires Node 8 or above**
+The documentation and source code for previous versions of the byu-jwt package are found on
+the [v3 branch](https://github.com/byu-oit/byu-jwt-nodejs/tree/v3) in this repository.
 
-## Table of Contents
+> **Note**
+> **Requires Node.js >= v18 *OR* a fetch polyfill such as [node-fetch](https://www.npmjs.com/package/node-fetch#providing-global-access).**
 
-- [Migrate from v1 to v2](#migrate-from-v1-to-v2)
-- [Migrate from v2 to v3](#migrate-from-v2-to-v3)
-- [API](#api)
-    - [Constructor](#constructor)
-    - [Authenticate](#authenticate)
-    - [Authenticate University API Middleware](#authenticate-university-api-middleware)
-    - [Decode JWT](#decode-jwt)
-    - [Get OpenID Configuration](#get-openid-configuration)
-    - [Get Pem](#get-pem)
-    - [Verify JWT](#verify-jwt)
-    - [Cache Time to Live](#cache-time-to-live)
-    - [Static Constants](#static-constants)
-- [Testing](#testing)
+# Contributing
 
-## Migrate from v1 to v2
+This project uses [Lerna](https://lerna.js.org) with [Nx](https://nx.dev) to build, test, and lint the source code.
+Please consult their documentation when making modifications to the maintenance process of this project.
 
-* Update to Node 8 or above
+There are a few commands that most of the packages share:
 
-## Migrate from v2 to v3
+- **build**: Compile the distribution code
+- **lint**: Lint the source code
+- **test**: Test the source code with [Ava](https://avajs.dev)
 
-* `getPublicKey` has
-  been [removed](https://github.com/byu-oit/byu-jwt-nodejs/commit/fe16edddd1f59a4f6c37acc29d9a20b5878626bd) - If you
-  were using it, look into the new `getPem` function
-* Ensure that the [`openssl`](https://nodejs.org/en/docs/meta/topics/dependencies/#openssl) shipped with your version of
-  Node supports the algorithms you need - We're now using that instead of expecting an `openssl` executable to be found
-  on the system.
-    * This is probably a non-issue because our JWTs have been using RSA-256, which `openssl` has supported for _years_.
+If you notice a problem, please submit an issue or create a PR with the fix!
 
-## API
+## Committing
 
-### Constructor
+Commit messages must adhere to
+the [angular conventional commit standard](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional#type-enum).
+[`commitlint`](https://github.com/conventional-changelog/commitlint) will enforce commit messages to follow this
+standard. Following a commit standard enables our distribution pipeline to publish new versions of each package
+automatically.
```

Comment:
I believe we can [have GitHub help us](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#metadata-restrictions) enforce conventional commits nowadays.

Reply:
That may be better than additional husky files in our repos

Reply:
We can do both.

In a hypothetical situation, supposing I have multiple commits I'm trying to merge (and since we don't often squash PRs here at OIT), I'd rather find out that my commit messages were bad in a pre-commit hook than from GitHub when I'm 5 commits in.

Reply:
True 👍🏻